src/content/quickstart/openshift/_index.md

```diff
@@ -6,8 +6,9 @@ weight: 20
 
 This quickstart guide covers the necessary steps to deploy two OpenShift Container Platform (OCP) clusters on AWS with
 full stack automation, also known as installer-provisioned infrastructure (IPI). Once the OpenShift clusters are deployed, we deploy
-Submariner to interconnect the two clusters. Note that this guide focuses on Submariner for network connectivity only. For Submariner with
-Service Discovery, please refer to the [Submariner with Service Discovery guide](./service_discovery/).
+Submariner with Service Discovery to interconnect the two clusters. Note that this guide focuses on Submariner deployment on clusters with
```

Comment:
There is some issue while rendering the index. Can you please take a look.
![Screenshot from 2020-11-05 13-53-22](https://user-images.githubusercontent.com/6574656/98216289-dd9f4b80-1f6e-11eb-9aa0-b09c2f07445a.png)

Reply:
@sridhargaddam yeah I've noticed that as well. If you roll over the "what seems to be just an empty space area" there, you will see that it actually has links to the shared pages, and you can even access them if you click on the URL. I am already looking at fixing this, but will do so in a different PR (note that this issue is seen with the current version of the website and is not something that this change has introduced).

Reply:
I see, thanks @nyechiel

Reply:
Yeah I had been wondering about this too, good job tracking down those details @nyechiel.